tv_apps/fling-player/js/fling_player.js

```diff
@@ -262,58 +286,28 @@
     this.setPlayButtonState('paused');
   };
 
-  proto._startUpdateControlPanelContinuously = function () {
-
-    if (this._contrlPanelUpdateTimer == null) {
-
-      // To stop this is important.
-      // We don't want they get messed with each other
-      this._stopSeekOnKeyPress();
-
-      this._contrlPanelUpdateTimer = setTimeout(
-        this._updateControlPanelContinuously.bind(this)
-      );
-      this.showControlPanel(true);
-    }
-  };
-
-  proto._stopUpdateControlPanelContinuously = function () {
-    clearTimeout(this._contrlPanelUpdateTimer);
-    this._contrlPanelUpdateTimer = null;
-  };
-
-  /**
-   * This is to keep auto updating the info and status on the control panel.
-   */
-  proto._updateControlPanelContinuously = function () {
-
-    if (this._contrlPanelUpdateTimer != null) {
-
-      var buf = this._player.getVideo().buffered;
-      var current = this._player.getRoundedCurrentTime();
-
-      this.writeTimeInfo('elapsed', current);
-      this.moveTimeBar('elapsed', current);
-
-      if (buf.length) {
-        this.moveTimeBar('buffered', buf.end(buf.length - 1));
+  proto._updateControlPanel = function () {
+    var i;
```

Comment:
declare this in first statement of for-loop is ok. What I mean is not to declare variables unrelated to for-loop condition in first statement of `for` loop. Please move declaration of `i` back to for-loop.